### DISCUSSION TITLE
/exception/npe.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#26] [Announcements]
# /exception/npe.html

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/exception/npe.html

**Comment by Yukkikwii** [comment]
笑死

**Comment by zebchn** [comment]
后续呢？
